packages/shared/src/components/profile/Header.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { CSSProperties, ReactElement } from 'react';
+import classNames from 'classnames';
+import { PublicProfile } from '../../lib/user';
+import ShareIcon from '../icons/Share';
+import SettingsIcon from '../icons/Settings';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/ButtonV2';
+import { useShareOrCopyLink } from '../../hooks/useShareOrCopyLink';
+import { ProfilePicture } from '../ProfilePicture';
+import { largeNumberFormat } from '../../lib/numberFormat';
+
+export type HeaderProps = {
+  user: PublicProfile;
+  isSameUser: boolean;
+  sticky?: boolean;
+  className?: string;
+  style?: CSSProperties;
+};
+
+export function Header({
+  user,
+  isSameUser,
+  sticky,
+  className,
+  style,
+}: HeaderProps): ReactElement {
+  const [, onShareOrCopyLink] = useShareOrCopyLink({
+    text: `Check out ${user.name}'s profile on daily.dev`,
+    link: user.permalink,
+    trackObject: () => ({ event_name: 'share profile', target_id: user.id }),
+  });
+
+  return (
+    <header
+      className={classNames('flex items-center px-4 h-12', className)}
+      style={style}
+    >
+      {sticky ? (
+        <>
+          <ProfilePicture user={user} nativeLazyLoading size="medium" />
+          <div className="flex flex-col ml-2 typo-footnote">
+            <div className="font-bold">{user.name}</div>
+            <div className="text-theme-label-tertiary">
+              {largeNumberFormat(user.reputation)} Reputation
+            </div>
```

Comment:
Do you need divs here or would `p` do to be more semantic as they only hold copy.

Reply:
It's not a `p` really but maybe a `span` would be better then. Right?

Reply:
To me text is most of the time `p` but not sure how others view it.
Span could also work, but I tend to use it for inline things because of it's native inline behaviour